it/controllers/CalculationControllerISpec.scala

```diff
@@ -487,6 +507,8 @@ class CalculationControllerISpec extends IntegrationSpecCommonBase with FeatureS
         penaltyStatus = LPPPenaltyStatusEnum.Accruing,
         penaltyAmountPaid = None,
         penaltyAmountOutstanding = Some(400.00),
+        penaltyAmountPosted = 0,
+        penaltyAmountAccruing = 0,
```

Comment:
This (penaltyAmountAccruing) would be 400, also I don't think it would have an outstanding amount anymore